runtime/particle-execution-context.js

```diff
@@ -161,9 +161,9 @@ export class ParticleExecutionContext {
             resolve(id);
           }}));
       },
-      createSlot: function(transformationParticle, transformationSlotName, hostedParticleName, hostedSlotName) {
```

Comment:
Is it OK to provide a null storeId? Documentation here or in particle might help.

Reply:
yes, it is nullable, added comment. and added a TODO to support multiple.
and renamed to handleId (because we call stores - handles on the particle side).